### PR TITLE
Refactor/bradcrumbs small screen

### PR DESCRIPTION
**Pull request by Aringoaway**
implement breadcrumbs in small screens

### CODE REVIEW

**Comment by Aringoaway** [comment]
@Pe5h4
breadcrumbs-small

File: src/styles/components/breadcrumbs.scss [hunk 1]
```diff
@@ -25,6 +25,55 @@
 
 @media screen and (max-width: 1024px) {
 	.breadcrumbs {
-		display: none;
+		display: inline-block;
+		text-align: left;
+		margin: auto 0;
+		h4 {
+			font-size: 20px;
```
Comment (Pe5h4) [inline src/styles/components/breadcrumbs.scss:32]: @Aringoaway Why did you change the font-size here from 24 to 20px?

File: src/styles/components/breadcrumbs.scss [hunk 2]
```diff
@@ -25,6 +25,55 @@
...
+		nav {
+			display: inline-block;
+		}
+	}
+}
+
+@media screen and (max-width: 800px) {
+	.breadcrumbs {
+		position: absolute;
+		top: 21px;
```
Comment (Pe5h4) [inline src/styles/components/breadcrumbs.scss:45]: @Aringoaway I am not sure about those paddings and margins to some very particular size

File: src/styles/components/breadcrumbs.scss [hunk 3]
```diff
@@ -25,6 +25,55 @@
...
+	}
+}
+
+@media screen and (max-width: 768px) {
+	.breadcrumbs {
+		display: inline-block;
+		text-align: left;
+		position: absolute;
+		bottom: -38px;
```
Comment (Pe5h4) [inline src/styles/components/breadcrumbs.scss:54]: @Aringoaway Here the same, not sure about this

File: src/styles/components/breadcrumbs.scss (hunk 2 above)
Comment (Pe5h4) [reply src/styles/components/breadcrumbs.scss:45]: I am pretty sure that it will break something else on some other screen

File: src/styles/index.scss [hunk 4]
```diff
@@ -150,6 +150,13 @@ code {
 		}
 	}
 }
+@media screen and (max-width: 768px) {
+	.header-sidebar-wrapper {
+		.padding-small-screens {
+			padding-top: 3rem!important;
```
Comment (Pe5h4) [inline src/styles/index.scss:156]: @Aringoaway 3rem is the height of the header?

**Review by Pe5h4 — CHANGES_REQUESTED** [review]

**Comment by Pe5h4** [comment]
@Aringoaway on some resolutuion, the padding top of the breadcrumbs is too much

File: src/styles/components/breadcrumbs.scss (hunk 1 above)
Comment (Aringoaway) [reply src/styles/components/breadcrumbs.scss:32]: @Pe5h4 because the font size is too large for a small screen

**Comment by Aringoaway** [comment]
@Pe5h4 can you review pls
breadcrumbs12
breacrumbs23

File: src/containers/Header/index.js [hunk 5]
```diff
@@ -43,7 +43,7 @@ export function Header(props) {
 							</Link>
 						</div>
 					}
-					<Breadcrumbs app={props.app}/>
+					{windowDimensions.width > 768 &&<Breadcrumbs app={props.app}/>}
```
Comment (Pe5h4) [inline src/containers/Header/index.js:46]: @Aringoaway You have no padding after `&&`

File: src/containers/Header/index.js [hunk 6]
```diff
@@ -31,7 +31,7 @@ export function Header(props) {
 		<header className={`application-header ${headerProperties ? 'header-props-open' : ""}`}>
 			{(props.app.props.hasSidebar || typeof props.app.props.hasSidebar == 'undefined') ?
 				<>
-					{windowDimensions.width < 800 &&
+					{windowDimensions.width < 769 &&
```
Comment (Pe5h4) [inline src/containers/Header/index.js:34]: @Aringoaway Out of curiosity - on line 46 you have this condition: 

https://github.com/TeskaLabs/asab-webui/blob/ab29a72cd4cb2029323e05bb785eb8a305d6eef1/src/containers/Header/index.js#L46

Why those numbers differ?

File: src/containers/Sidebar/index.js [hunk 7]
```diff
@@ -63,16 +63,16 @@ const Sidebar = (props) => {
 	const toggle = () => setModal(!modal);
 
 	return (
-		windowDimensions.width < 768 ?
+		windowDimensions.width < 769 ?
```
Comment (Pe5h4) [inline src/containers/Sidebar/index.js:66]: @Aringoaway Why `769` and not `768` ? I mean everywhere in this PR

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
@Aringoaway Pls add changelog as well and I have another question: is sidebar still collapsable on larger screens?

**Comment by Aringoaway** [comment]
@Pe5h4 I corrected all the mistakes, can you review it again

File: src/containers/Sidebar/SidebarBottomItem.js [hunk 8]
```diff
@@ -42,6 +42,13 @@ const SidebarBottomItem = ({ item, sidebarLogo }) => {
 		});
 	}
 
+	if (screenWidth && screenWidth < 800) {
```
Comment (Pe5h4) [inline src/containers/Sidebar/SidebarBottomItem.js:45]: @Aringoaway Why is here `800` when in some other files you have `<= 768` ?

File: src/styles/components/breadcrumbs.scss [hunk 9]
```diff
@@ -25,6 +25,54 @@
...
+	.breadcrumbs {
+		margin: auto 0 0;
+	}
+}
+
+@media screen and (max-width: 768px) {
+	.breadcrumbs {
+		display: inline-block;
+		text-align: left;
+		//position: absolute;
```
Comment (Pe5h4) [inline src/styles/components/breadcrumbs.scss:52]: @Aringoaway Can you pls remove those commented lines? If there is no `TODO:` written, I suppose those comments are redundant and it can be removed, right?

File: src/styles/index.scss [hunk 10]
```diff
@@ -150,6 +153,17 @@ code {
...
 }
+@media screen and (max-width: 768px) {
+	.header-sidebar-wrapper {
+		.app-body.without-sidebar {
+			.main.padding-small-screens {
+				padding-top: 0 !important;
+				margin: 0 1rem;
+			}
+		}
+
```
Comment (Pe5h4) [inline src/styles/index.scss:164]: @Aringoaway Pls remove this empty line

File: src/styles/components/sidebar.scss [hunk 11]
```diff
@@ -243,6 +243,29 @@ $sidebar-item-font-size: 14px;
...
+		}
+	}
+	.app-sidebar {
+		.sidebar-nav {
+			.nav {
+				.sidebar-item-button {
+					&.mobile-collapsed {
+						display: none;
+					}
+
```
Comment (Pe5h4) [inline src/styles/components/sidebar.scss:261]: @Aringoaway Pls remove this empty line

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
